homeassistant/components/persistent_notification/__init__.py

```diff
@@ -140,6 +140,15 @@ def async_dismiss(hass: HomeAssistant, notification_id: str) -> None:
     )
 
 
+@callback
+@bind_hass
```

Comment:
Bind hass is legacy and we don’t use it for new calls

Reply:
done